### PR TITLE
Switch to using more official browser brand icons 

### LINKED ISSUE #5921: Switch to using more official browser brand icons
### Current behavior:

Currently we use font awesome 5 free brand icons for browsers. 

**Our dropdown**

Screen Shot 2019-12-10 at 10 12 54 AM

**FA available icons**

Screen Shot 2019-12-10 at 10 09 51 AM

### Desired behavior:

This choice of icons has some downsides:

1. No separate icon for 'Electron' - using Chrome for Electron browser is likely confusing.
2. No separate icon for Canary and Chromium, we just have to do static colors.
3. They don't look as nice as rasterized images would. 
4. They don't keep up with browser branding changes as fast as we could.

Since browsers are a key part of our product, these should probably look nicer. 

Screen Shot 2019-12-10 at 10 14 39 AM

### Versions

3.7.0

### To contribute

- Tests for browser icon dropdown: https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/cypress/integration/project_nav_spec.js#L120:L120
- Code for browser icon dropdown icons: 
	- jsx: https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/src/project-nav/browsers.jsx#L73:L73
	- browser model: https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/src/lib/browser-model.js#L29:L29
	- helper utility (this is used for other parts of the desktop-gui, so should not be removed/changed) https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/src/lib/utils.js#L28:L28
- Browser icons css: https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/src/app/nav.scss#L172:L172

### PR DESCRIPTION
**Pull request by hiramzamorano**
- Closes  #5921 

### User facing changelog
We replaced the browser icons in the Test Runner with more official browser brand icons.

### How has the user experience changed?
Before
Screen Shot 2019-12-17 at 6 54 27 PM

After

Screen Shot 2020-01-09 at 10 27 52 PM

### PR Tasks

- [x] Have tests been added/updated?

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@hiramzamorano Thanks for the contribution! Could you please sign our CLA? https://github.com/cypress-io/cypress/pull/5991#issuecomment-566832665

**Comment by hiramzamorano** [comment]
Hi @jennifer-shehane, thanks for the feedback, i'm going to fix those issues :)

File: packages/desktop-gui/cypress/integration/project_nav_spec.js [hunk 1]
```diff
@@ -122,20 +122,6 @@ describe('Project Nav', function () {
...
-            })
-
-            let family = dropdownBrowsers[i].family
-
-            family = family === 'electron' ? 'chrome' : family
-
-            // first one is shown in selection, so skip first
-            cy.wrap($i).should('have.class',
-              `fa-${family}`)
-          })
```
Comment (jennifer-shehane) [inline packages/desktop-gui/cypress/integration/project_nav_spec.js]: This test needs to be added back and updated to ensure all the svg icons are displaying within the browser list.

File: packages/desktop-gui/src/project-nav/browsers.jsx [hunk 2]
```diff
@@ -58,19 +67,44 @@ export default class Browsers extends Component {
...
+        }
+        case 'firefox': {
+          icon = firefox
+          break
+        }
+        default: {
+          icon = defaultBrowser
+          break
+        }
+      }
```
[the remark this answers is not in the document]
Comment (hiramzamorano) [reply packages/desktop-gui/src/project-nav/browsers.jsx]: Hi @jennifer-shehane , I made a small code refactor, creating the getIcon function to keep the code clean, I'm not sure if it's something you were looking for, any suggestion or recommendation is welcome :D. 
I also made an alignment of the icons, sorry I missed that detail in the previous pull request.

File: packages/desktop-gui/cypress/integration/project_nav_spec.js (hunk 1 above)
Comment (jennifer-shehane) [reply packages/desktop-gui/cypress/integration/project_nav_spec.js]: Sorry, maybe I have not been clear. The test needs to iterate through `this.config.browsers` (like it did before) for each icon and ensure that the class matches the `this.config.browsers.name`.

 We do not need a test that '.browser-icons' is a class, but that the exact icon is being displayed for the browser that should be shown in the list. So for each item in the dropdown it should have class `.chrome`, `.chromium`, `.canary`, etc. in order, removing the selected browser.

Comment (hiramzamorano) [reply packages/desktop-gui/cypress/integration/project_nav_spec.js]: Hi again @jennifer-shehane , first of all i'm sorry for the misunderstanding, i hope that this time i have understood well not to spend your time more hahaha, i'm trying to give my best and all your recommendations are helping me :D
If there is any other improvement that you see convenient to do just tell me :)

**Comment by hiramzamorano** [comment]
You're right @jennifer-shehane , we can improve the default generic icon to make it look better.
I can suggest you these options.

1)Screen Shot 2020-01-03 at 1 48 10 PM

2)Screen Shot 2020-01-03 at 1 46 33 PM

3)Screen Shot 2020-01-03 at 1 39 55 PM

4)Screen Shot 2020-01-03 at 1 38 40 PM

5)Screen Shot 2020-01-03 at 1 45 11 PM

6)Screen Shot 2020-01-03 at 1 42 40 PM

What do you think?

**Comment by jennifer-shehane** [comment]
@hiramzamorano I prefer #5. Thanks for providing the options. 👍

**Comment by hiramzamorano** [comment]
It seems that the previous commit had no effect on the icon, that's weird because I was testing it with the steps you tell me.
The solution is to center the svg image in its own container, I made some small changes hoping that now if they are reflected.

This should be the end result.

Screen Shot 2020-01-09 at 10 27 52 PM

**Comment by brian-mann** [comment]
The electron logo should not be electron - it should be dark Chrome. Our users don't care what the version of Electron is, they care about the version of Chromium that electron ships with.

**Comment by jennifer-shehane** [comment]
@hiramzamorano The team decided to revert this commit per these comments. https://github.com/cypress-io/cypress/pull/6141#issue-361579111 

You're welcome to open a new PR. I'm sorry for the outcome of this. No one expressed their differing opinions from the team until after it was merged unfortunately.

**Pull request merged**